frontend/src/contexts/AuthenticationContext.js

```diff
@@ -25,7 +25,7 @@ const AuthenticationContextProvider = ({ children }) => {
     const isAuthenticated = isConnected && !isWrongNetwork && address === authenticatedAddress;
 
     useEffect(() => {
-        if (isWrongNetwork) switchNetwork(primaryChain.id)
+        if (isWrongNetwork) switchNetwork?.(primaryChain.id)
```

Comment:
Proper way of doing this is:

`if (isAuthenticated && isWrongNetwork)`

Reply:
Tried that, still was crashing with "switchNetwork is not a function"

Reply:
Yeah I mean I'm no Wagmi expert I don't know the actual library, I just meant please let's ignore the awful syntax and approach that they suggest; it is very bad